src/index.ts

```diff
@@ -1,6 +1,6 @@
 import './polyfills'
```

Comment:
Need to update the readme. 
Also. How to pass an agent options with example: keep-alive: true

Reply:
There wont be such option. I want it to use it by default.